### PR TITLE
Revamp empty states

### PR DESCRIPTION
**Pull request by timja**
See https://github.com/jenkinsci/branch-api-plugin/pull/240

<details>
<summary>Organisation</summary>

image

</details>

<details>
<summary>Multibranch</summary>

image

</details>

cc @fqueiruga @dwnusbaum @bitwiseman

### CODE REVIEW

**Review by fqueiruga — APPROVED** [review]

File: src/main/resources/org/jenkinsci/plugins/workflow/multibranch/WorkflowBranchProjectFactory/getting-started.jelly [hunk 1]
```diff
@@ -25,7 +25,4 @@
 <st:compress xmlns:j="jelly:core" xmlns:st="jelly:stapler" xmlns:f="/lib/form">
   Pipeline Branch projects support building branches within a repository containing a pipeline script. Per default it uses
```
Comment (timja) [inline src/main/resources/org/jenkinsci/plugins/workflow/multibranch/WorkflowBranchProjectFactory/getting-started.jelly:26]:
```suggestion
  Pipeline Branch projects support building branches within a repository containing a pipeline script. By default it uses
```

**Comment by timja** [comment]
@bitwiseman @dwnusbaum @car-roll this is now ready as the upstream PR in branch-api is released

File: pom.xml [hunk 2]
```diff
@@ -66,17 +66,16 @@ THE SOFTWARE.
         <revision>2.25</revision>
         <changelist>-SNAPSHOT</changelist>
         <java.level>8</java.level>
-        <jenkins.version>2.222.4</jenkins.version>
+        <jenkins.version>2.277.1</jenkins.version>
```
Comment (bitwiseman) [inline pom.xml:69]:
```suggestion
        <jenkins.version>2.277.4</jenkins.version>
```

File: pom.xml [hunk 3]
```diff
@@ -115,6 +110,7 @@ THE SOFTWARE.
         <dependency>
             <groupId>org.jenkins-ci.plugins</groupId>
             <artifactId>branch-api</artifactId>
+            <version>2.6.4</version>
```
Comment (bitwiseman) [inline pom.xml:113]: 2.6.4 is in the bom.
```suggestion
```

File: src/main/resources/org/jenkinsci/plugins/workflow/multibranch/WorkflowMultiBranchProjectFactory/getting-started-links.jelly [hunk 4]
```diff
@@ -0,0 +1,19 @@
...
+    </li>
+    <li class="content-block">
+        <a href="https://www.jenkins.io/doc/book/pipeline/multibranch/" class="content-block__link content-block__help-link">
+            <span>Creating Multibranch Projects</span>
+            <span class="trailing-icon">
+                <l:svgIcon class="icon-sm" href="${imagesURL}/material-icons/svg-sprite-content-symbol.svg#ic_link_24px" />
+            </span>
+        </a>
+    </li>
+</st:compress>
```
Comment (bitwiseman) [inline src/main/resources/org/jenkinsci/plugins/workflow/multibranch/WorkflowMultiBranchProjectFactory/getting-started-links.jelly:19]: End with newline?

**Review by bitwiseman — APPROVED** [review]
Minor tweaks.

**Comment by timja** [comment]
@bitwiseman 🚢 ?

File: src/main/resources/org/jenkinsci/plugins/workflow/multibranch/WorkflowMultiBranchProjectFactory/getting-started-links.jelly (hunk 4 above)
Comment (timja) [inline src/main/resources/org/jenkinsci/plugins/workflow/multibranch/WorkflowMultiBranchProjectFactory/getting-started-links.jelly:19]:
```suggestion
</st:compress>

```

**Comment by timja** [comment]
release?

**Pull request merged**